assets/app/view/game/hex.rb

```diff
@@ -44,7 +44,7 @@ def render
           when :track, :token_or_track
             opaque = @round.connected_hexes[@hex]
             clickable ||= opaque
-          when :token, :route, :home_token
+          when :token, :route, :home_token, :move_token_due_to_ambigious_city
```

Comment:
can we change the name to be :reposition_token or :move_token

Reply:
Reposition sounds better, but I'll leave the action as move (1882 has a move private)